.golangci.next.reference.yml

```diff
@@ -106,7 +106,6 @@ linters:
     - stylecheck
     - tagalign
     - tagliatelle
-    - tenv
```

Comment:
I noticed that the repository for [`tenv`](https://github.com/sivchari/tenv) is not archived yet and seems supported. Could you please clarify why we are deprecating it in favor of the relatively new `usetesting`? I was under the impression that we deprecate a linter only if it is no longer archived or supported. Examples: #1755, #3125, #4436, #4652, #4916 etc.

Additionally, does this mean that we might also consider deprecating `cyclop`, `funlen`, `lll`, and a few other linters, since their functionality is duplicated by some `revive` rules?

Reply:
because:
- there is redundancy between tenv and usetesting
- usetesting detects more `os.Setenv` cases
- cleaning before v2

Reply:
@sivchari FYI

Reply:
> because:
> - there is redundancy between tenv and usetesting
> - usetesting detects more `os.Setenv` cases

This PR implicitly suggests to others to create a new linter, improve it, and deprecate the old one. However, a better approach is to create a PR and improve the existing linter.

Reply:
> This PR implicitly suggests to others to create a new linter, improve it, and deprecate the old one. However, a better approach is to create a PR and improve the existing linter.

No, this PR implicitly suggests nothing.
usetesting is not an improvement of tenv.
usetesting handles more things than `t.env`/`os.env`